### ISSUE TITLE
Sympa not loading some wwsympa.conf variables after system restart

### ISSUE THREAD

**Issue by qosobrin** [sympa-community/sympa#198]
I am having some weird problem with both my preproduction and production installations of Sympa. We are running sympa from Jessie's Debian package that happens to be version 6.1.23

Our installation is very straightforward, we have only on robot running in the default location debian places it (/etc/sympa) and we made very little customizations, all of them related to removing some scenarios and modifying four web tt2  templates. The server is running smoothly, since at the moment we only have a handful of lists.

The problem aroused when we tried to rebuild the web archive of one list. We used the web interface for that at http://sympa.domain.org/wws/serveradmin/archives and the command was accepted as it was reflected in the log file. No more output in the log file other than the following (no confirmation that the archives were rebuilt nor the contrary):

```
Feb 16 19:25:44 sympa wwsympa[3197]: info [robot lists.domain.org] [session 53515855095691] [client 192.168.23.18] [user listmaster@domain.org] main::do_serveradmin() do_serveradmin
Feb 16 19:25:46 sympa wwsympa[3197]: info [robot lists.domain.org] [session 53515855095691] [client 192.168.23.18] [user listmaster@domain.org] main::do_serveradmin() do_serveradmin
Feb 16 19:25:50 sympa wwsympa[3197]: info [robot lists.domain.org] [session 53515855095691] [client 192.168.23.18] [user listmaster@domain.org] [list atic-probas] main::do_rebuildarc() do_rebuildarc(atic-probas, )
Feb 16 19:25:50 sympa wwsympa[3197]: info [robot lists.domain.org] [session 53515855095691] [client 192.168.23.18] [user listmaster@domain.org] [list atic-probas] main::do_rebuildarc() File: /var/spool/sympa/outgoing/.rebuild.atic-probas@lists.domain.org
Feb 16 19:25:50 sympa wwsympa[3197]: info [robot lists.domain.org] [session 53515855095691] [client 192.168.23.18] [user listmaster@domain.org] [list atic-probas] main::do_admin() do_admin
```

After we restarted sympa service:

```
service sympa restart
```

the log file appears full of errors:

```
Feb 16 19:30:25 sympa bounced.pl: info Configuration file read, default log level 0
Feb 16 19:30:25 sympa bounced[5943]: notice bounced Started
Feb 16 19:30:25 sympa bulk[5933]: info Configuration file read, default log level 0
Feb 16 19:30:25 sympa bulk[5933]: info Starting bulk master daemon, pid 5948
Feb 16 19:30:25 sympa bulk[5948]: notice bulk.pl 6.1.23 Started
Feb 16 19:30:25 sympa sympa[5932]: info Configuration file read, default log level 0
Feb 16 19:30:25 sympa archived.pl: info Configuration file read, default log level 0
Feb 16 19:30:25 sympa archived[5954]: notice archived 6.1.23 Started
Feb 16 19:30:25 sympa task_manager[5962]: info Configuration file read, default log level 0
Feb 16 19:30:25 sympa sympa[5932]: info Server started for all, pid 5963
Feb 16 19:30:25 sympa sympa[5963]: notice Sympa 6.1.23 started
Feb 16 19:30:31 sympa archived[5954]: notice main::rebuild() Rebuilding  with M2H_ADDRESSMODIFYCODE :
Feb 16 19:30:31 sympa archived[5954]: notice main::rebuild() Rebuilding atic-probas@lists.domain.org archive completely
Feb 16 19:30:31 sympa archived[5954]: err tools::copy_dir() Directory source '/var/lib/sympa/arc//atic-probas@lists.domain.org/2014-11/arctxt' doesn't exist. Copy impossible
Feb 16 19:30:31 sympa archived[5954]: err Archive::clean_archive_directory() Unable to create a temporary directory where to store files for HTML escaping (). Cancelling.
Feb 16 19:30:32 sympa archived[5954]: err tools::copy_dir() Directory source '/var/lib/sympa/arc//atic-probas@lists.domain.org/2010-02/arctxt' doesn't exist. Copy impossible
Feb 16 19:30:32 sympa archived[5954]: err Archive::clean_archive_directory() Unable to create a temporary directory where to store files for HTML escaping (). Cancelling.
Feb 16 19:30:32 sympa archived[5954]: err tools::copy_dir() Directory source '/var/lib/sympa/arc//atic-probas@lists.domain.org/2009-11/arctxt' doesn't exist. Copy impossible
Feb 16 19:30:32 sympa archived[5954]: err Archive::clean_archive_directory() Unable to create a temporary directory where to store files for HTML escaping (). Cancelling.
```

Suddenly it is complaining that the arctxt files do not exist. It is trying to read those files in the 
/var/lib/sympa/arc/ directory, which happens to be the default value for the arc_path variable, even when we have it correctly defined in our wwsympa.conf file:

```
arc_path        /var/lib/sympa/wwsarchive
```
Even wwsympa knows the variable is correctly defined as it is correctly shown in http://sympa.domain.org/wws/edit_config, but the archiving module seems to be "confused".

```
parameter name  value                      apply to          default
arc_path        /var/lib/sympa/wwsarchive  lists.domain.org  /var/lib/sympa/arc
```

We have tried to restart sympa service, with no luck, the problem continues. Even when we rebooted the server the problem continued. The most weird thing is that, after a couple of hours, when we rebooted the server again it began to behave without problem. 

Anybody has come to such a strange behavior? Any idea of what I could try?

Sorry if this site does not support sympa 6.1.x but I didn't know any other place to post my question.

Thank you in advance. Regards.

**Comment by ikedas** [comment]
@qosobrin, though it might not help, I think you would be better to check the recent version of Sympa. The jessie is obsoleted stable of Debian and sympa-6.1 series is previous stable of upstream.

**Comment by qosobrin** [comment]
I am finally giving a try to the version included in Debian Stretch. As far as I could test this bug is not included in that 6.2.x version.

Thank you very much.
Best regards.
